src/main/java/kitchenpos/products/tobe/domain/ProductPrice.java

```diff
@@ -0,0 +1,35 @@
+package kitchenpos.products.tobe.domain;
+
+import javax.persistence.Access;
+import javax.persistence.AccessType;
+import javax.persistence.Embeddable;
+import java.math.BigDecimal;
+import java.util.Objects;
+
+@Embeddable
+@Access(AccessType.FIELD)
+public class ProductPrice {
+
+    private final BigDecimal price;
+
+    public ProductPrice() {
+        price = null;
+    }
+
+    public ProductPrice(BigDecimal price) {
```

Comment:
`domain` 패키지 외에 사용하는 곳이 없어 보이네요. public이 아닌 package-private으로 바꾸면 어떨까요?

```suggestion
    ProductPrice(BigDecimal price) {
```